content/php/concepts/string-functions/terms/htmlentities/htmlentities.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: 'htmlentities()'
+Description: 'Converts all applicable characters to HTML entities.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`htmlentities()`** function converts characters to HTML entities. To convert HTML entities back to characters, use the **[html_entity_decode()](../html-entity-decode/html-entity-decode.md)** function.
+
+## Syntax
+
+```pseudo
+htmlentities(string,flags,encoding,double_encode)
+```
+
+The `htmlentities()` function has one required parameter and three optional parameters:
```

Comment:
@SSwiniarski I have some solutions like below and wonder what the most suitable solution for this case is.

+Should I just provide information about the string parameter, a little bit about $flag, $encoding, $double_encode then lead users into find more information from php.net website? 

+Should I just specify 3-4 constant name with description for the $flag parameter, and 3-4 Charset with Aliases and description for the $encoding parameter? I just feel like the information is too long and I want to lead users to outside resources like w3school, or php.net for more information

+Should I keep all the details like I just wrote and just write the description in my way so that users can understand them easier?

Reply:
@phuongtrieu97coder keep what you have and rewrite to the language isn't a copy of another website. You can trim down the list of constants, but we don't like linking to outside documentation in Docs.